tests/projects/idl/test_rpc_noserver/xmake.lua

```diff
@@ -0,0 +1,8 @@
+add_rules("mode.debug", "mode.release")
+
+target("idltest_rpc_noserver")
+    set_kind("binary")
+    --add_rules("midl")
```

Comment:
?

Reply:
>  --add_rules("midl")
this line is part of an old version of the test, I will remove it